pyodide_build/mkpkg.py

```diff
@@ -10,6 +10,12 @@
 PACKAGES_ROOT = Path(__file__).parent.parent / 'packages'
 
 
+def get_sdist_url_entry(json_content):
+    for entry in json_content['urls']:
+        if entry['filename'].endswith('.tar.gz'):
+            return entry
+
```

Comment:
We should raise an exception here if it didn't find anything suitable.

Reply:
OK, I force-pushed a new version that raises an exception.